src/types/StopLoss.sol

```diff
@@ -51,11 +53,33 @@ contract StopLoss is BaseConditionalOrder {
         returns (GPv2Order.Data memory order)
     {
         Data memory data = abi.decode(staticInput, (Data));
-        (, int256 latestSellPrice,,,) = data.sellTokenPriceOracle.latestRoundData();
-        (, int256 latestBuyPrice,,,) = data.buyTokenPriceOracle.latestRoundData();
+        // scope variables to avoid stack too deep error
+        {
+            (, int256 basePrice,,uint256 sellUpdatedAt,) = data.sellTokenPriceOracle.latestRoundData();
```

Comment:
naming is correct, just since not everyone reading this code may be familiar with base/quote, I thought it was easier to keep the sell/buy nomenclature, but no strong feeling.

Reply:
Sure, no problems. I think from a readability perspective, having the quote / base nomenclature is suitable when dealing with the chainlink oracles, given the example for normalising prices on their side references base / quote.